ember-server/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -43,6 +43,11 @@ private[server] object ServerHelpers {
   private val serverFailure =
     Response(Status.InternalServerError).putHeaders(org.http4s.headers.`Content-Length`.zero)
 
+  private def durationToFinite(duration: Duration): Option[FiniteDuration] = duration match {
+    case f: FiniteDuration => Some(f)
+    case _ => None
+  }
```

Comment:
I think I've seen this twice now.

Reply:
I did I can put it somewhere more common. Ideas?

Reply:
Just in `internal/package.scala` maybe? It's not a big deal. It's just four lines.